js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -825,18 +825,18 @@ membershipUserLdapAttributeHelp=Used just if Membership Attribute Type is UID. I
 samlCapabilityConfig=SAML capabilities
 accessTokenSignatureAlgorithmHelp=JWA algorithm used for signing access tokens.
 derFormatted=DER formatted
-periodicChangedUsersSyncHelp=Whether periodic synchronization of changed or newly created LDAP users to Keycloak should be enabled or not
-signatureAlgorithmHelp=The signature algorithm to use to sign documents. Note that 'SHA1' based algorithms are deprecated and can be removed in the future. It is recommended to stick to some more secure algorithm instead of '*_SHA1'
+periodicChangedUsersSyncHelp=Whether periodic synchronization of changed or newly created LDAP users to Keycloak should be enabled or not.
+signatureAlgorithmHelp=The signature algorithm to use to sign documents. Note that 'SHA1' based algorithms are deprecated and can be removed in the future. It is recommended to stick to some more secure algorithm instead of '*_SHA1'.
 allow-default-scopes.tooltip=If on, newly registered clients will be allowed to have client scopes mentioned in realm default client scopes or realm optional client scopes
 emailVerified=Email verified
-addExecutionHelp=Execution can have a wide range of actions, from sending a reset email to validating an OTP
+addExecutionHelp=Execution can have a wide range of actions, from sending a reset email to validating an OTP.
 requestObjectRequiredHelp=Specifies if the client needs to provide a request object with their authorization requests, and what method they can use for this. If set to "not required", providing a request object is optional. In all other cases, providing a request object is mandatory. If set to "request", the request object must be provided by value. If set to "request_uri", the request object must be provided by reference. If set to "request or request_uri", either method can be used.
 clientScopesRolesScope=If there is no role scope mapping defined, each user is permitted to use this client scope. If there are role scope mappings defined, the user must be a member of at least one of the roles.
 passwordPoliciesHelp.notUsername=The password cannot match the username.
 passwordPoliciesHelp.notContainsUsername=The password cannot contains the username.
 removeConfirm_one=Are you sure you want to remove this group
 createUserProviderSuccess=User federation provider successfully created
-countHelp=Specifies how many clients can be created using the token
+countHelp=Specifies how many clients can be created using the token.
```

Comment:
This is not a complete sentence.  I'm not sure if it's proper to end with a period.